packages/node_modules/@node-red/editor-api/lib/editor/module.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright JS Foundation and other contributors, http://js.foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+var apiUtils = require("../util");
+var runtimeAPI;
+
+module.exports = {
+    init: function(_runtimeAPI) {
+        runtimeAPI = _runtimeAPI;
+    },
+    createSubflow: function(req,res) {
+        const body = req.body;
+        const meta = body ? body.meta : null;
+        const flow = body ? body.flow : null;
+        runtimeAPI.module.createSubflow(meta, flow).then(function(result) {
```

Comment:
Wrong message signature for the runtimeAPI.

I see you haven't added a matching file under `runtime/lib/api`. Any communication between `editor-api` and `runtime` must go via `runtime/lib/api/*`.

All functions on the runtimeAPI take a single `opts` argument - have a look at the others to see what I mean.

Reply:
Fixed the argument to `opts`. Also, added code to `runtime/lib/api`.